index.html

```diff
@@ -20,19 +20,21 @@
         <nav class="nav-top">
```

Comment:
- [ ] [OPTIONAL] You can add some ```margin``` and change the color of the close button on your design so it can be the same as the Figma one
- [ ] [OPTIONAL] Your design has an additional line in the middle of the menu, kindly remove because it's not on the design

- Your design:
![Screenshot_20220511_052824](https://user-images.githubusercontent.com/25227022/167769486-9094945a-43fc-4e6a-bc0b-98499ee648a8.png)

- Figma design:
![Screenshot_20220511_052842](https://user-images.githubusercontent.com/25227022/167769517-0f2fb5ea-e6da-475b-891c-1978180c8547.png)